File: library_architecture/src/main/java/com/cs/library_architecture/utils/LogUtils.java
package com.cs.library_architecture.utils;

import android.util.Log;

/**
 * author : ChenSen
 * data : 2019/6/21
 * desc:
 */
public class LogUtils {

    public static void logi(String msg) {
        Log.i("tag", msg);
    }

    public static void log(String msg) {
        Log.e("tag", msg);
    }
}

File: common/src/main/java/com/cs/library_architecture/utils/LogUtils.kt
package com.cs.library_architecture.utils

import android.util.Log

object LogUtils {

    private const val TAG = "tag"

    fun log(msg: String) {
        Log.e(TAG, msg)
    }

    fun log(tag: String, msg: String) {
        Log.e(tag, msg)
    }

    fun logd(msg: String) {
        Log.d(TAG, msg)
    }

    fun logd(tag: String, msg: String) {
        Log.d(tag, msg)
    }

}